how-to-make-a-release

```diff
@@ -0,0 +1,2 @@
+1) Update vendor and test
```

Comment:
s/vendor/version/

Reply:
Ah vendor is the content directory as this go so we bundle all the dependencies and their versions.